exercises/1-the-manual-menace/README.md

```diff
@@ -163,7 +163,7 @@ https://codeready-workspaces.apps.<DOMAIN_FOR_YOUR_CLASS>/dashboard/#/load-facto
 
 7. Let's add two more params dicts to pass to our template to be able to create a `dev` and `test` project. At the top of `enablement-ci-cd/inventory/host_vars/projects-and-policies.yml` create a dictionary called `dev` and `test` similar to how you see `ci_cd` defined.
 
- * In your editor; Open `enablement-ci-cd/inventory/host_vars/projects-and-policies.yml` and add the following:
+ * In your editor, open `enablement-ci-cd/inventory/host_vars/projects-and-policies.yml` and add the following lines before `openshift_cluster_content`:
```

Comment:
I believe Ansible runs in order, so I guess it's important to define the variables values before using them... I might be wrong though.